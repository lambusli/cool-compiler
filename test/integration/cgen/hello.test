class Main {
  main():Object {
    (new IO).out_string("hello, world!\n")
  };
};
