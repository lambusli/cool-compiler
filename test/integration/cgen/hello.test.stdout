hello, world!
COOL program successfully executed
